Cargo.toml

```diff
@@ -46,7 +46,7 @@ strum_macros = "0.21"
 sha2 = "0.9"
 hex = "0.4"
 zip = "0.5.11"
-qrcodegen = "1"
```

Comment:
I'm not sure I like this. While I get why you're doing this, `qrcodegen` seems to be well-maintained while `qrcode` appears to be pretty dead. Probably QR code stuff doesn't need a lot of maintenance but I'm still a bit unhappy about this. Any chance to make `qrcodegen` do the work?

Reply:
See for instance https://github.com/nayuki/QR-Code-generator/blob/master/rust/examples/qrcodegen-demo.rs.

Reply:
that would be low

> Any chance to make `qrcodegen` do the work?

I'll give it a try!